.github/workflows/stable-spec-tests.yml

```diff
@@ -71,8 +71,9 @@ jobs:
           extract: true
       - name: Clone execution-spec-tests and consume tests
         run: |
-          git clone https://github.com/ethereum/execution-spec-tests -b ${{ env.FIXTURES_TAG }} --depth 1
+          git clone https://github.com/ethereum/execution-spec-tests
           cd execution-spec-tests
+          git checkout 250a064ba38cd92cad02691f4f7c2ecbefedd954
```

Comment:
What I'm doing here is use a specific commit to download the repo to use the fixed `consume direct` binary now. 

Whenever we cut a new release in `executions-spec-tests`, we can revert this and just update the `FIXTURE_TAG`. But until then, we can have a useful CI failing when it should. 

We'll probably cut a new release soon as soon as I can go over all the failing tests exposed by the witness assertions. So this change won't last that long.

Reply:
could we not use tags for this, so that we have to update the tag instead of committing a new hash each time?

Reply:
ah yeah ok, I thought that was a tag from the other repo, my bad.